src/ontology/uberon-edit.obo

```diff
@@ -134848,7 +134849,7 @@ id: UBERON:0009865
 name: Hatschek's pit
 def: "A deep ciliated fossa on the dorsal midline of the buccal cavity (the region of the gut behind the mouth). Among other things, it secrets mucus which entraps food particles from the water[WP]." [Wikipedia:Hatschek's_pit]
 xref: Wikipedia:Hatschek's_pit
-is_a: UBERON:0006846 ! surface groove
+is_a: UBERON:0003102 ! surface structure
```

Comment:
Not human anatomy and have not found a detailed reference. With that said, I suggest modelling to align with other fossa.

Reply:
fossa has both material and immaterial terms too lol, please not me to fix that, traumatized by this >.<
But similar to Endostyle, it secretes mucus, so it has to be material in my mind